### DISCUSSION TITLE
Language code for Taiwanese transcriptions

### DISCUSSION THREAD

**Discussion by meg261995** [openai/whisper#2230] [Q&A]
Does whisper support Taiwanese transcriptions? If so,  Could someone help me out with what language code to use here? Chinese/Mandarin/Cantonese?

**Comment by glangford** [comment]
See
* https://github.com/openai/whisper/discussions/25

The full set of language codes supported by whisper is here:
https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/tokenizer.py#L10

**Comment by jake1271** [comment]
Chinese is a bit of a mess on Whisper which seems to randomly mix simplified and traditional. You can try adding a prompt to request simplified Mandarin. Hopefully they'll release a new language code for traditional Mandarin in the future.

**Comment by laubonghaudoi** [comment]
@meg261995 By Taiwanese do you mean Southern Min in Taiwan, code `nan-tw`? Whisper doesn't support that at the moment.

**Comment by kalvinchang** [comment]
Empirically, ChatGPT (which uses Whisper) seems to support Taiwanese but translates it to Mandarin(?

![image](https://github.com/user-attachments/assets/b6d89caa-ec3e-4dff-872b-7f67a74034ca)

I asked in Taiwanese:

lí thiann ū bô guá teh kóng sánn
你聽有無我咧講啥
